### PR TITLE
Jinen branch1

### PR DESCRIPTION
**Pull request by jinendradhurvey**
completed the code and edited the suggestion please review for merge

### CODE REVIEW

File: Blue_Green_Ami.pkr.hcl [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+build {
+  name = "blue-build"
+  sources = [
+    "source.amazon-ebs.Blue-web-server"
+  ]
+
+  provisioner "ansible" {
+    playbook_file = "./main.yml"
+    extra_arguments = ["--extra-vars", "@./files/c_blue.yml"]
```
Comment (pasc-ed) [inline Blue_Green_Ami.pkr.hcl]: Variable files should not be in the `files` folder.

**Review by pasc-ed — CHANGES_REQUESTED** [review]
Only need to change the folder of the variable files. Well done for the filters !

File: Blue_Green_Ami.pkr.hcl (hunk 1 above)
Comment (jinendradhurvey) [reply Blue_Green_Ami.pkr.hcl]: changes the variable file name.

**Review by pasc-ed — APPROVED** [review]
LGTM

**Pull request merged**